doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/managing-iot-data.md

```diff
@@ -1 +1,558 @@
+time=2016-09-29T11:12:35.923Z | lvl=DEBUG | corr=7c675d9d-e62b-4f67-99f7-e82063ad671a | trans=7c675d9d-e62b-4f67-99f7-e82063ad671a | op=IoTAgentNGSI.MongoDBGroupRegister | srv=tourguide | subsrv=/Franchise1 | msg=Looking for entity params ["resource","apikey"] | comp=IoTAgent
+Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
+time=2016-09-29T11:12:35.930Z | lvl=DEBUG | corr=7c675d9d-e62b-4f67-99f7-e82063ad671a | trans=7c675d9d-e62b-4f67-99f7-e82063ad671a | op=IoTAgentNGSI.MongoDBGroupRegister | srv=tourguide | subsrv=/Franchise1 | msg=Device group for fields [["resource","apikey"]] not found: [{"resource":"/iot/d","apikey":"tourguide-devices"}] | comp=IoTAgent
+time=2016-09-29T11:12:35.935Z | lvl=DEBUG | corr=7c675d9d-e62b-4f67-99f7-e82063ad671a | trans=7c675d9d-e62b-4f67-99f7-e82063ad671a | op=IoTAgentNGSI.MongoDBGroupRegister | srv=tourguide | subsrv=/Franchise1 | msg=Storing device group with id [57ecf7235755b6010061870a], type [Restaurant], apikey [tourguide-devices] and resource [/iot/d] | comp=IoTAgent
+time=2016-09-29T11:12:35.946Z | lvl=DEBUG | corr=7c675d9d-e62b-4f67-99f7-e82063ad671a | trans=7c675d9d-e62b-4f67-99f7-e82063ad671a | op=IoTAgentNGSI.IOTAMService | srv=tourguide | subsrv=/Franchise1 | msg=response-time: 38 | comp=IoTAgent
+```
+This information will be stored in a MongoDB database.  We can check this with:
+```
+$ docker exec -i -t mongodb mongo
+MongoDB shell version: 2.6.11
+connecting to: test
+Welcome to the MongoDB shell.
+For interactive help, type "help".
+For more comprehensive documentation, see
+	http://docs.mongodb.org/
+Questions? Try the support group
+	http://groups.google.com/group/mongodb-user
+```
+
+We can list the available databases and select the one used by the IoT Agent, `iotagentul`.
+```
+> show dbs;
+admin   	(empty)
+iotagentul  0.031GB
+local   	0.031GB
+orion   	0.031GB
+> use iotagentul;
+switched to db iotagentul
+```
+In this database we have two collections, `devices` and `groups`.  The service we've just registered should be in the `groups` collection.
+```
+> show collections;
+devices
+groups
+system.indexes
+> db.groups.find();
+{ "_id" : ObjectId("57ecf7235755b6010061870a"), "subservice" : "/Franchise1", "service" : "tourguide", "type" : "Restaurant", "apikey" : "tourguide-devices", "resource" : "/iot/d", "staticAttributes" : [ ], "__v" : 0 }
+```
+As we can see, the service has been stored.  Next is to register the sensors.
+
+### Register a new sensor with the IoT Agent
+
+To register a new temperature sensor, we need to send a POST request to `http://localhost:4041/iot/devices` with the following JSON payload:
+```
+{
+	"devices": [
+    	{
+        	"device_id": "DEV_ID",
```

Comment:
instead of using here the placeholder "DEV_ID" or "ENTITY_ID", we can use directly the concrete strings and explain below what they refer to. It will  make it easier to experiment by copy pasting